### ISSUE TITLE
Error: Could not locate linux data

### ISSUE THREAD

**Issue by techwebspot** [Tomas-M/linux-live#163]
I made the live bootable usb and script executed properly but still I can't boot when my usb filesystem had ext4 and owner & group is owned by root user. 

And when second time I made the live bootable usb. My usb filesystem was ntfs and owner & group is owned by normal user. Here I want to specify one more thing that in this case I mount the iso file and then copy into usb (rather than making zip with the script and then unzip) and run the script which is in boot directory. And then this time I can boot but get "Could not locate linux data" error. 

So I don't know where I'm going wrong. Is there any specific requirement like we need NTFS filesystem only? And we can't boot when owner and group is owned by root? Directly mounting the iso rather than making zip and unzip it in usb creates the problem of "Could not locate linux data" error?

PS: The generated iso works like a charm.

**Comment by techwebspot** [comment]
> For EXT4, you need to use the Extlinux bootloader to make it work properly, It works in the same way as Syslinux, with slight changes to make it bootable with this filesystem.
> 
> About NTFS on bootable USBs: be aware that some motherboards have BIOSes that doesn't recognize bootable devices built on NTFS, it's more common than you might think.
> 
> If you want to ensure compatibility when using the included Syslinux bootloader on the kit, stick to the old FAT32, even with the known limitation of this FS on single file size (4GB).

Yeah I heard FAT32 is most reliable but my build zip file is of something 4.8 GB so I can't use FAT32 because of limitation.

Okay to use Extlinux can you please guide me a bit what I need to do exactly after unzipping the zip file content in usb file? 

And the bootable usb which I made works in legacy mode and not UEFI. Is there any way to make it work in UEFI too? 

As I told I'm getting Could not locate linux data error, is it because I mount iso and copy files in usb rather than making a zip? 

Is there anything with permission like I told earlier? Because in first trial it didn't work when owner and group is owned by root. And in second trial it works when owner and group is owned by normal. It works because in second trial it have NTFS or it works because of permission? 

And yeah sorry for asking too many questions. I know it's preposterous but making bootable and all takes so much time that's why I can't do much experiment.

**Comment by nonvwvnon** [comment]
my filesystem is formatted in ext4 and did the unzip and run the  bootinst. the usb ssd drive give a 'Error: Could not locate linux data' error.

on another occassion, i once change the usb cable and the usb ssd drive is able to boot up the OS..
it work on usb thumbdrive.

it is a 'hit and miss' situation.

i wonder if there is a solution for th above problem.
